### PR TITLE
Landmine markdown file

### PR DESCRIPTION
**Pull request by Mr-Maximus**
..

### CODE REVIEW

File: projects/LandMine.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+<!--Describe who will be your project's users-->
+General public interested in buying and selling property.
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+N/A
+
+## Github repo
+<!--Attach a link to your GitHub repo if it's OSS-->
+https://github.com/Mr-Maximus/landmine_project
```
Comment (autonome) [inline projects/LandMine.md:59]: The source code must be licensed MIT. Please add license file to your repo.

File: projects/LandMine.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+### Team Size
+2 members
+
+### Team members
+[Madan Raj.S](https://github.com/Mr-Maximus)
+[Mukesh Jaiswal](https://github.com/MukeshJaiswal01)
+
+
+## How the community can engage
+GitHub Discussion:N/A
```
Comment (autonome) [inline projects/LandMine.md:82]: Discussion link is required here, you can follow the instructions here: https://github.com/ipfs/community#ecosystem-projects

File: projects/LandMine.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+## Project Description
+<!--Describe your project in a few sentences. -->
+Landmine is a platform where each property {like apartments,bunglow} is represented as an nft and then these nft are put up for sale.
+-Buyers can buy the properties listed on the site.
+-After Buyer agress to buy the property they will upload the agreement details which will be saved in the ipfs platform and seller will complete the transaction after the completion of payment.
+-Buyer can verify these transaction details by using their property id by entering their id and they will get the ipfs link of their agreement.
+
+### Use of IPFS
+- We have used ipfs for storing the documents related to the agreement details .
```
Comment (autonome) [inline projects/LandMine.md:17]: Can you share a link to this? I did not see in the Github repo or links in this document.

**Review by autonome — CHANGES_REQUESTED** [review]
Some fixes required, noted inline.

File: projects/LandMine.md (hunk 2 above)
Comment (Mr-Maximus) [reply projects/LandMine.md]: i have made the necessary changes here is the link 
<a href="https://github.com/ipfs/community/discussions/517">GitHub Discussion</a>

File: projects/LandMine.md (hunk 3 above)
Comment (Mr-Maximus) [reply projects/LandMine.md:17]: l have updated it now. here is the <a href="https://gateway.ipfs.io/ipfs/QmTiKQjNEAeNpxdkMNvWpKj4jbz8cRaEp4EjQAyAHYURAT">link</a>

File: projects/LandMine.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+### Team Size
+2 members
+
+### Team members
+[Madan Raj.S](https://github.com/Mr-Maximus)
+[Mukesh Jaiswal](https://github.com/MukeshJaiswal01)
+
+
+## How the community can engage
+GitHub Discussion:https://github.com/ipfs/community/discussions/517
```
Comment (autonome) [inline projects/LandMine.md:84]: This is discussion for different project, can you add one for yours?

**Review by autonome — CHANGES_REQUESTED** [review]
needs a link update

File: projects/LandMine.md (hunk 4 above)
Comment (Mr-Maximus) [reply projects/LandMine.md]: Sorry my bad . I have updated it now . here is the <a href="https://github.com/ipfs/community/discussions/529">link</a>

**Pull request merged**